plugins/capture.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: capture
+spec:
+  version: v0.1.0
+  platforms:
+  - selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+    uri: https://github.com/sysdiglabs/kubectl-capture/releases/download/v0.1.0/kubectl-capture.v0.1.0.tar.gz
+    sha256: "86c947a9fa9acb73eeb179e29ede20755a426a4fb96aa82375246a356f3a804f"
+    bin: "kubectl-capture"
+  shortDescription: >-
+    Triggers a Sysdig capture to troubleshoot the running pod
+  homepage: https://github.com/sysdiglabs/kubectl-capture
+  caveats: |
+    To analyze the capture file you will need to use Sysdig Inspect. You can use it as docker image directly:
```

Comment:
This caveats read more like a short command help. It would be better to move this into a place where the user expects it, like `kubectl capture --help`.

Instead, I would expect more true caveats, such as required pre-requisites to run this plugin. For example, you need to be able to run privileged pods.